.buildkite/scripts/test_e2e.sh

```diff
@@ -46,11 +46,11 @@ ${test_runner_binary} \
     ${BUILDKITE:+--basedir ${TEST_BASE_DIR:-$PWD}/e2e} \
     --basedir.no_cleanup \
     --e2e.node.binary ${node_binary} \
-    --e2e.client.binary_dir ${WORKDIR}/target/default/debug \
-    --e2e.runtime.binary_dir ${WORKDIR}/target/${runtime_target}/debug \
-    --e2e.runtime.loader ${WORKDIR}/target/default/debug/oasis-core-runtime-loader \
-    --e2e.tee_hardware ${OASIS_TEE_HARDWARE:-""} \
-    --remote_signer.binary ${WORKDIR}/go/oasis-remote-signer/oasis-remote-signer \
+    --e2e/runtime.client.binary_dir ${WORKDIR}/target/default/debug \
+    --e2e/runtime.runtime.binary_dir ${WORKDIR}/target/${runtime_target}/debug \
+    --e2e/runtime.runtime.loader ${WORKDIR}/target/default/debug/oasis-core-runtime-loader \
+    --e2e/runtime.tee_hardware ${OASIS_TEE_HARDWARE:-""} \
+    --remote-signer.binary ${WORKDIR}/go/oasis-remote-signer/oasis-remote-signer \
```

Comment:
In case of a command-line argument it should be `remote_signer` (as it was before) to be consistent with other arguments.

Reply:
This is a form of  `--<testname>.<parameter>` and `remote-signer` is the name of the test, so I would leave this as it is.